lightning/src/ln/interactivetxs.rs

```diff
@@ -1662,14 +1662,55 @@ impl InteractiveTxConstructor {
 	}
 }
 
+/// Determine whether a change output should be added or not, and if so, of what size,
+/// considering our given inputs, outputs, and intended contribution.
+/// Computes and takes into account fees.
+#[allow(dead_code)] // TODO(dual_funding): Remove once begin_interactive_funding_tx_construction() is used
+pub(super) fn need_to_add_funding_change_output(
```

Comment:
nit: could this perhaps be named `calculate_funding_change_output_value` or even `calculate_change_output_value`? seems like it currently hints at returning a `bool`.